netbox/netbox/settings.py

```diff
@@ -502,6 +502,9 @@ def _setting(name, default=None):
 MAINTENANCE_EXEMPT_PATHS = (
     f'/{BASE_PATH}admin/',
     f'/{BASE_PATH}extras/config-revisions/',  # Allow modifying the configuration
+    LOGIN_URL,
+    LOGIN_REDIRECT_URL,
+    LOGOUT_REDIRECT_URL
```

Comment:
Do the redirect URLs need to be included? Only the actual login view should need to write to the database.

Reply:
I added these thinking maybe it'll be required for 3rd party auth integrations

Reply:
Yeah that's fair, good call.